packages/core/src/modules/credentials/__tests__/V1CredentialService.propose.offer.test.ts

```diff
@@ -1,7 +1,7 @@
 import type { AgentConfig } from '../../../agent/AgentConfig'
```

Comment:
It's getting quite crazy with the dots. Should we create a `V1CredentialService.proposal.test.ts` or should we maybe just call it `V1CredentialService.proposal-offer.test.ts` Ideally we remove the dots all together, it's more meant as a file type, while here it's just part of the filename IMO

Reply:
Files renamed